edenia/antelope-tools
# Fix/ricardian contract link 1378

### Fix ricardian contract link and add libre node types

### What does this PR do?

- Update eoscr-components package
- Fix ricardian contracts links
- Resolve #1378
- Resolve #1388
- Resolve #1402

### Steps to test

1. Run the project locally
2. Go to Ricardian contract page
3. Check the links are correct
4. Go to bp.json generator page
5. Check the styles and new types for libre

### Screenshots

# Ricardian contract page

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/db68c5d5-9eb7-4497-941d-b12e31b30346)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/ce0e54d0-42c1-4c48-a4f0-1318f1f5cf0d)

# Generator page

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/8eff5e87-5944-41fc-a8f2-85db4c1179c5)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/20496880-09ad-4f61-84c8-63c991590a92)

# Nodes page

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/d4fb60dd-f6de-4285-bd54-e8d88ee968bc)
